attributes/default.rb

```diff
@@ -3,7 +3,20 @@
 # Attributes for InfluxDB
 
 # By default, always installs the latest by specifying nil
-default['influxdb']['version'] = nil
+# NOTE: whether it is a safe choice to use nil for version because we cannot be sure if the latest version is supported fully by this cookbook. See `elasticsearch` cookbook for more information
+default['influxdb']['version'] = '0.13.0'
+default['influxdb']['install_type'] = 'package'
+
+default['influxdb']['download_urls'] = {
+  'debian' => 'https://dl.influxdata.com/influxdb/releases',
+  'rhel' => 'https://dl.influxdata.com/influxdb/releases'
+}
+
+# platform_family keyed download sha256 checksums
+default['influxdb']['checksums']['0.10.3-1']['debian'] = '106e4884d8c654f1e43b803b21acc86e6494cafc50a46bcdb4330d0a51bd7aea'
```

Comment:
`0.10.3.1` older shasums?  can we make the default be for what is being set at the top `['influxdb']['version'] = '0.13.0'`